site/src/docs/components/login/api.mdx

```diff
@@ -0,0 +1,768 @@
+| `getState`                             | State of the client. See <InternalLink size="M" href="/components/login/modules#states">Oidc client states</InternalLink>.                                                                                                                                                                                                                                               | `NO_SESSION`                                                                                              |
+| `getToken`                             | Returns a token. The function is async because token renewal might be in progress and the current token might be outdated after that. If renewal was in progress but failed, then the old token is returned. Use `isRenewing()` and the exported function `getUserToken(user, tokenType)` to get the token synchronously and safely. The Promise will never be rejected. | `string` or `undefined`                                                                                   |
+| `getUser`                              | Returns a user object or null.                                                                                                                                                                                                                                                                                                                                           | <InternalLink href="/components/login/api#object-types-from-oidc-client-ts">User</InternalLink> or `null` |     |
+| `getUserManager`                       | Returns the UserManager.                                                                                                                                                                                                                                                                                                                                                 | <InternalLink href="/components/login/api#object-types-from-oidc-client-ts">UserManager</InternalLink>    |     |
+| `handleCallback`                       | Handles the callback data and returns a user object - if the user is valid.                                                                                                                                                                                                                                                                                              | `Promise<User>` or `Promise<OidcClientError>`                                                             |
```

Comment:
In the description part you could link to OidcClientError and User object.

Reply:
Link to User is now added. OidcClientError is link to docs, not to source code.